### PR TITLE
virsh nodecpustats: add negative case.

### PR DESCRIPTION
**Pull request by Chenditang**
test invalid cpu number.

### CODE REVIEW

File: libvirt/tests/cfg/virsh_cmd/host/virsh_nodecpustats.cfg [hunk 1]
```diff
@@ -12,9 +12,13 @@
...
             libvirtd = "on"
         - unexpect_option:
-            virsh_cpunodestats_options = "xyz"
             status_error = "yes"
             libvirtd = "on"
+            variants:
+                - invalid_option:
+                    virsh_cpunodestats_options = "--xyz"
+                - invalid_cpuNum:
+                    virsh_cpunodestats_options = "--cpu 11111111"
```
Comment (waynesun09) [inline libvirt/tests/cfg/virsh_cmd/host/virsh_nodecpustats.cfg]: is 11111111 a boundary value or a random number?

Comment (Chenditang) [inline libvirt/tests/cfg/virsh_cmd/host/virsh_nodecpustats.cfg]: just a random number, test Invalid cpuNum.
$  virsh nodecpustats --cpu 111111
error: Unable to get node cpu stats
error: Invalid cpuNum in linuxNodeGetCPUStats

Comment (waynesun09) [inline libvirt/tests/cfg/virsh_cmd/host/virsh_nodecpustats.cfg]: Ok

**Comment by waynesun09** [comment]
ACK since 11111111 is a big enough value for current testing, and libvirt just use host cpu number with int type for this.

**Comment by PandaWei** [comment]
Hi,
I think  @waynesun09 's consideration is reasonable.
It's more better that to confirm the boundary value and set --cpu to "boundary +1".

**Comment by PandaWei** [comment]
@Chenditang 
Do you have any idea to confirm the boundary value ?
if there's no good way to get the boundary,
I think  11111111 is also acceptable.
Please attach the result, And let's pass this PR.

**Comment by will-Do** [comment]
Yes, a random negative value is OK.

**Comment by Chenditang** [comment]
The result is running on libvirt-1.1.3.9-1.fc20.x86_64

\# ./run -k --no-downloads -t libvirt --tests virsh.nodecpustats
SETUP: PASS (0.20 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/autotest/client/tests/virt/logs/run-2015-07-21-17.38.20/debug.log
TESTS: 4
(1/4) type_specific.io-github-autotest-libvirt.virsh.nodecpustats.no_option: PASS (34.73 s)
(2/4) type_specific.io-github-autotest-libvirt.virsh.nodecpustats.unexpect_option.invalid_option: PASS (12.31 s)
(3/4) type_specific.io-github-autotest-libvirt.virsh.nodecpustats.unexpect_option.invalid_cpuNum: PASS (12.44 s)
(4/4) type_specific.io-github-autotest-libvirt.virsh.nodecpustats.with_libvirtd_stop: PASS (17.41 s)
TOTAL TIME: 76.93 s (01:16)
TESTS PASSED: 4
TESTS FAILED: 0
SUCCESS RATE: 100.00 %

**Comment by Chenditang** [comment]
@PandaWei @will-Do 
Updated, Thank you for your suggestion.
